tests/asserts/final/publication/internal.py

```diff
@@ -150,6 +151,12 @@ def correct_values_in_detail(workspace, publ_type, name, *, exp_publication_deta
         '_thumbnail': {'path': f'/layman_data_test/workspaces/{workspace}/{publ_type_dir}/{name}/thumbnail/{name}.png'},
         'access_rights': {'read': ['EVERYONE'], 'write': ['EVERYONE']},
         'image_mosaic': False,
+        '_table_uri': ConnectionString(
+            url='postgresql://docker:docker@postgresql:5432/layman_test',
+            schema=workspace,
+            table=f'layer_{uuid}',
```

Comment:
It's not raw UUID, see get_table_name in layer.db or https://github.com/LayerManager/layman/pull/748/files#diff-dd775c3c3bcba2016f947d1dc08aa9492a0f6ef61f5c30141ba283048ea0307aR218